app/controllers/incident/IncidentCodeController.scala

```diff
@@ -35,42 +36,34 @@ import scala.concurrent.{ExecutionContext, Future}
 class IncidentCodeController @Inject() (
   override val messagesApi: MessagesApi,
   implicit val sessionRepository: SessionRepository,
-  navigatorProvider: IncidentNavigatorProvider,
+  @IdentificationDetails implicit val navigator: Navigator,
```

Comment:
Any reason this is not using the IncidentNavigatorProvider?

Reply:
Yea it's a bit of an anomaly here. I had made the change to use the IncidentNavigatorProvider but my last commit made me resolve conflicts between my local and remote branches. It's either deleted the change or I've omitted it during the conflicts being resolved. Sorted now :+1: